low-level/framework/utils/salt_util.py

```diff
@@ -160,7 +161,8 @@ def get_consul_port(self, _err=None):
         try:
             if self.pillar_info is not None:
                 data_store_config = \
-                    self.pillar_info[self.node_id][self.SSPL_KEY][self.DATASTORE_KEY]
+                    self.pillar_info[self.node_id][self.CLUSTER_KEY][self.node_id]\
```

Comment:
@mohitpathak1 this change is not required, I had specifically asked provisioner team about this and I was told that this change is not required.

Reply:
Okay, will revert this.